06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/output/README.md

```diff
@@ -0,0 +1,9 @@
+#### Output
+- **Using**: Heroku
+- **Deliver**: live URLs to a `review` app, your `staging` & `production` apps
+- **With**: *independently*
+
+#### Result
+- [Review URL](https://inverdex-staging-pr-8.herokuapp.com/)
+- [Staging URL](https://inverdex-staging.herokuapp.com/)
+- [Production URL](https://inverdex.herokuapp.com/)
```

Comment:
There's an open pull request on it so I'm still expecting _another_ review. That's the reason the production link is failing.